src/Symfony/Component/AssetMapper/Tests/CompiledAssetMapperConfigReaderTest.php

```diff
@@ -54,8 +55,9 @@ public function testLoadConfig()
     public function testSaveConfig()
     {
         $reader = new CompiledAssetMapperConfigReader($this->writableRoot);
-        $this->assertEquals($this->writableRoot.\DIRECTORY_SEPARATOR.'foo.json', realpath($reader->saveConfig('foo.json', ['foo' => 'bar'])));
-        $this->assertEquals(['foo' => 'bar'], json_decode(file_get_contents($this->writableRoot.'/foo.json'), true));
+
+        $this->assertSame(Path::normalize($this->writableRoot.\DIRECTORY_SEPARATOR.'foo.json'), Path::normalize(realpath($reader->saveConfig('foo.json', ['foo' => 'bar']))));
```

Comment:
Why normalizing paths here ? Comparing 2 real paths should not require any normalization.

Reply:
`CompiledAssetMapperConfigReader` does some internal operations on the directory string (in `saveConfig()`) which leads to a path with mixed `/` and `\`.

Reply:
But the assertion is not on the return value of `saveConfig`. It is on the result of `realpath` for that return value.

Reply:
It doesn't @xabbuh, I had a look at an outdated Appveyor log. Closing this PR, thanks for the review!